patterns/base/button/pattern.yml

```diff
@@ -4,8 +4,8 @@ twig: ./button.twig
 sass: ./button.scss
 category: base
 data:
+  class: base--button
   buttons:
-    class: base--button
     basic:
```

Comment:
please put the button versions under a `versions` title like this:
https://github.com/pattern-library/pattern-library/blob/master/patterns/message/pattern.yml
note though that the `versions` section in messages shouldn't be under `content` though, it should be on the same level as `content`